app/pages/Pdf/ServicePdfPage.tsx

```diff
@@ -111,14 +130,13 @@ export const ServicePdfPage = () => {
           <Loader />
         </>
       )}
-      ;
       {!pdfSource && (
         <div
           ref={ref}
           className={`serviceHandoutPdf ${styles.handoutInvisible}`}
         >
           {/* The below styles contain some overrides of global .renderedMarkdown and table tag selector styles */}
-          <style>
+          <style translate="no" ref={styleRef}>
```

Comment:
I'm laughing a little because I'm imagining how you discovered that we need `translate="no"` here 😂.